src/wp-includes/default-filters.php

```diff
@@ -110,6 +110,10 @@
 	add_action( $action, 'wp_maybe_update_user_counts', 10, 0 );
 }
 
+// Sticky posts
```

Comment:
```suggestion
// Sticky posts.
```

It would be good to avoid this is possible but I suspect it isn't.